OSDOCS-15988 [NETOBSERV] Module Aditi tool rules: TaskStep (#98466)
openshift/openshift-docs: modules/network-observability-enriched-flows.adoc

You can send network flows to Kafka, IPFIX, the Red{nbsp}Hat build of OpenTelemetry, or all three at the same time. For Kafka or IPFIX, any processor or storage that supports those inputs, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data. For OpenTelemetry, network flow data and metrics can be exported to a compatible OpenTelemetry endpoint, such as Red{nbsp}Hat build of OpenTelemetry, Jaeger, or Prometheus.

> After configuration, network flows data can be sent to an available output in a JSON format. For more information, see "Network flows format reference".

Reviewer: I know this was just moved around and not new, but it's actually not necessarily JSON (for instance, IPFIX is not JSON). Maybe we can just keep the ref to the flow format:

```suggestion
For more information about the output format, see "Network flows format reference".
```
